isthmus/src/test/java/io/substrait/isthmus/SimplePlansTest.java

```diff
@@ -24,6 +24,11 @@ public void filter() throws IOException, SqlParseException {
     assertProtoPlanRoundrip("select * from lineitem WHERE L_ORDERKEY > 10");
   }
 
+  @Test
+  public void in() throws IOException, SqlParseException {
+    assertProtoPlanRoundrip("select * from lineitem WHERE L_ORDERKEY IN (10, 20)");
+  }
```

Comment:
This test ends up creating the following Calcite plan
```
LogicalProject(L_ORDERKEY=[$0], L_PARTKEY=[$1], L_SUPPKEY=[$2], L_LINENUMBER=[$3], L_QUANTITY=[$4], L_EXTENDEDPRICE=[$5], L_DISCOUNT=[$6], L_TAX=[$7], L_RETURNFLAG=[$8], L_LINESTATUS=[$9], L_SHIPDATE=[$10], L_COMMITDATE=[$11], L_RECEIPTDATE=[$12], L_SHIPINSTRUCT=[$13], L_SHIPMODE=[$14], L_COMMENT=[$15])
  LogicalFilter(condition=[OR(=($0, 10), =($0, 20))])
    LogicalTableScan(table=[[LINEITEM]])
```

So while this is a useful test, it doesn't test the SEARCH path.

Reply:
We're testing that we handle the in() function instead of crashing in this test.